### ISSUE TITLE
unauthorized projects showing up in my deploy list

### ISSUE THREAD

**Issue by Rob46** [GoogleCloudPlatform/google-cloud-eclipse#3654]
- Cloud Tools for Eclipse version: 1.8.4.202002191941
- Google Cloud SDK version: 334.0.0 (non-managed)
- Eclipse version: 4.14.0.v20191210-0610
- OS: Windows 10 10.0
- Java version: 1.8.0_301

**What did you do?
When I now go to deploy an app I see projects that shouldn't be in the list:
![image](https://user-images.githubusercontent.com/25462429/129111086-bf8cd4bd-f1fe-473d-b805-4cf1e0859b48.png)
There was one bogus one before ("brave-watch") and now there is another ("you-can-see-this-project").
This shouldn't be happening. Google has some kind of permission problem that allows any random project to show up in my projects list. 

**What did you expect to see?**
Only my projects.

**What did you see instead?**
My projects + 2 bogus projects.

**Comment by dzou** [comment]
Hi @Rob46, thanks for the report.

> This shouldn't be happening. Google has some kind of permission problem that allows any random project to show up in my projects list.

Are you sure that those projects are not test projects created by someone in your organization? It is possible to see projects that you think are unrelated to you but were created to include you as a member by someone else that you might not have been aware of.

**Comment by Rob46** [comment]
Absolutely certain not projects created by someone else in our organization - its only me ;-)

RE "It is possible to see projects that you think are unrelated to you but were created to include you as a member by someone else that you might not have been aware of." 
**Not within my organization**... I am guessing some random company could have specified our project id by accident/typo?
In the google cloud console for our organizatoin we see this:
![image](https://user-images.githubusercontent.com/25462429/129271174-4fb396f1-0f1f-45bc-93b0-7ae7d470498b.png)

And for no organization we see this:
![image](https://user-images.githubusercontent.com/25462429/129271233-a1c4e0bd-cba1-48ee-ab14-de72ff684e5e.png)
I have no idea where those projects came from.

**Comment by dzou** [comment]
Ah I see. This is probably an issue with random projects adding you as a member for some reason. 

I would suggest opening a [Support Case in the Cloud Console](https://console.cloud.google.com/support), they can help figure out what is going on with those project permissions. This is outside the scope of the Eclipse extension since we are just displaying whatever projects are returned by the API.

**Comment by vbresan** [comment]
@Rob46 Have you resolved this problem? It looks like I have been "assigned" to same projects.

**Comment by elharo** [comment]
Hmm looks like I no longer have permissions to close issues in this repo? @briandealwis @chanseokoh  If there's a bug here, it's in the API and on the server side, not in the plugin, and we can't address it in this repo.

**Comment by Rob46** [comment]
I rolled back to an earlier version of eclipse.

**Comment by chanseokoh** [comment]
As shown in https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/3654#issuecomment-897974918 where the project shows up on the Google Cloud Console, the issue is unrelated to this Eclipse plugin. As mentioned in https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/3654#issuecomment-897990832, please open a support case for investigation.

**Comment by ericwolter** [comment]
Only for future searchees (like me):
TLDR: Unsubscribe your account from "google-appengine@googlegroups.com" group.

I have posted the solution to the stackoverflow question.
https://stackoverflow.com/questions/70690669/projects-under-no-organization-that-cannot-be-accessed/74411798#74411798
